os/arch/arm/Kconfig

```diff
@@ -104,6 +119,49 @@ config ARCH_CHIP
 	default "amebad"	if ARCH_CHIP_AMEBAD
 	default "amebalite"	if ARCH_CHIP_AMEBALITE
 	default "stm32h745"     if ARCH_CHIP_STM32H745
+	default "imx6"		if ARCH_CHIP_IMX6
+
+config ARM_THUMB
+        bool "Thumb Mode"
+        default n
+
+config ARM_HAVE_DPFPU32
+        bool
+        select ARCH_HAVE_DPFPU
+        default n
+        ---help---
+                FPU implemented in the VFPv[3|4]-D32 format that supports
+                32 double-precision floating-point registers.
+
+config ARM_HAVE_NEON
+        bool
+        default n
+        select ARM_HAVE_DPFPU32
+        ---help---
+                Decide whether support NEON instruction
```

Comment:
Minor: "Decide whether to support NEON instruction" 
OR "Decide whether the architecture supports NEON instruction"